src/Domain/Migrations/Version20201005215932.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Migrations;
+
```

Comment:
```suggestion

use App\Domain\Migrations\TablesNameCatalog;
```
I think the pre-commit hook github with php cs fixer remove this statement, but i think we need it to use it line 13.